Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```diff
@@ -194,30 +194,18 @@
     drawdepth: Mobs
     sprite: Mobs/Silicon/Bots/cleanbot.rsi
     state: cleanbot
+  - type: Drain
+    range: 1
+    unitsDestroyedPerSecond: 6
   - type: Construction
     graph: CleanBot
     node: bot
   - type: SentienceTarget
     flavorKind: station-event-random-sentience-flavor-mechanical
-  - type: Absorbent
-    pickupAmount: 10
-  - type: UseDelay
-    delay: 2
-  - type: SolutionRegeneration
-    solution: absorbed
-    generated:
-      reagents:
-        - ReagentId: Water
-          Quantity: 10
-  - type: SolutionPurge
-    solution: absorbed
-    preserve:
-      - Water
-    quantity: 10
   - type: SolutionContainerManager
     solutions:
-      absorbed:
-        maxVol: 50
+      drainBuffer:
+        maxVol: 30
   - type: MovementSpeedModifier
```

Comment:
Please don't just delete master in merge conflicts.

Reply:
this is resolved. Yea sorry i didnt really know what to do my master was a mess and i know you was trying to help me but it all made things worse and then nobody could provide me with any real answers.